bundles/binding/org.openhab.binding.zwave/database/remotec/zts110.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Product>
+	<Model>ZTS-110</Model>
+	<Label lang="en">Remotec Z-Wave Touchscreen Programmable Thermostat</Label>
+	<CommandClasses>                  <!-- Z-Wave Supported Command Classes: -->
+		<Class><id>0x20</id></Class>  <!-- COMMAND_CLASS_BASIC -->
+		<Class><id>0x27</id></Class>  <!-- COMMAND_CLASS_SWITCH_ALL -->
+		<Class><id>0x31</id></Class>  <!-- COMMAND_CLASS_SENSOR_MULTILEVEL -->
+		<Class><id>0x40</id></Class>  <!-- COMMAND_CLASS_THERMOSTAT_MODE -->
+		<Class><id>0x42</id></Class>  <!-- COMMAND_CLASS_THERMOSTAT_OPERATING_STATE -->
+		<Class><id>0x43</id></Class>  <!-- COMMAND_CLASS_THERMOSTAT_SETPOINT -->
+		<Class><id>0x44</id></Class>  <!-- COMMAND_CLASS_THERMOSTAT_FAN_MODE -->
+		<Class><id>0x45</id></Class>  <!-- COMMAND_CLASS_THERMOSTAT_FAN_STATE -->
+		<Class><id>0x72</id></Class>  <!-- COMMAND_CLASS_MANUFACTURER_SPECIFIC -->
+		<Class><id>0x85</id></Class>  <!-- COMMAND_CLASS_ASSOCIATION -->
+		<Class><id>0x86</id></Class>  <!-- COMMAND_CLASS_VERSION -->
+	</CommandClasses>
+	<Configuration>
+		<Parameter>
+			<Index>1</Index>
+			<Type>int</Type>
+			<Default>2</Default>
+			<Minimum>1</Minimum>
+			<Maximum>4</Maximum>
+			<Size>1</Size>
+			<Label lang="en">Swing</Label>
+			<Help lang="en"><![CDATA[Set Swing value from 1F/.5C to 4F/2.0C Available settings: 1 - 4]]> </Help>
+		</Parameter>
+		<Parameter>
+			<Index>2</Index>
+			<Type>int</Type>
+			<Default>2</Default>
+			<Minimum>1</Minimum>
+			<Maximum>4</Maximum>
+			<Size>1</Size>
+			<Label lang="en">Differntial</Label>
+			<Help lang="en"><![CDATA[Set Differential value from 1F/.5C to 4F/2.0C Available settings: 1 - 4]]> </Help>
+		</Parameter>
+		<Parameter>
+			<Index>3</Index>
+			<Type>int</Type>
+			<Default>500</Default>
+			<Minimum>500</Minimum>
+			<Maximum>4000</Maximum>
+			<Size>1</Size>
```

Comment:
Happy to fix it. Do I make changes to the file and resubmit in same branch
or do I need to close this pull and make changes and then recommit in new
branch with new pull?
On Jul 1, 2015 2:00 PM, "Chris Jackson" <notifications@github.com> wrote:

> In bundles/binding/org.openhab.binding.zwave/database/remotec/zts110.xml
> <https://github.com/openhab/openhab/pull/2838#discussion_r33712102>:
>
> > +			<Index>2</Index>
> > +			<Type>int</Type>
> > +			<Default>2</Default>
> > +			<Minimum>1</Minimum>
> > +			<Maximum>4</Maximum>
> > +			<Size>1</Size>
> > +			<Label lang="en">Differntial</Label>
> > +			<Help lang="en"><![CDATA[Set Differential value from 1F/.5C to 4F/2.0C Available settings: 1 - 4]]> </Help>
> > +		</Parameter>
> > +		<Parameter>
> > +			<Index>3</Index>
> > +			<Type>int</Type>
> > +			<Default>500</Default>
> > +			<Minimum>500</Minimum>
> > +			<Maximum>4000</Maximum>
> > +			<Size>1</Size>
>
> we need to get this right though. The fact that we can’t find another
> thermostat doesn’t help anyway since other thermostats will be different
> with different configuration… Looking at the manual, I would change these
> params to 2 bytes long
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openhab/openhab/pull/2838/files#r33712102>.
>